Makefile

```diff
@@ -14,7 +14,7 @@ OPERATORCOURIER := $(shell command -v operator-courier 2> /dev/null)
 LICENSEFINDERBINARY := $(shell command -v license_finder 2> /dev/null)
 DEPENDENCY_DECISION_FILE = $(PROJECT_PATH)/doc/dependency_decisions.yml
 MANIFEST_RELEASE ?= 1.0.$(shell git rev-list --count master)
-APPLICATION_REPOSITORY_NAMESPACE ?= apicastoperatormaster
+APPLICATION_REPOSITORY_NAMESPACE ?= apicastoperatorcommunity
```

Comment:
no need to change this, we are not pushing metadata to application registry

Reply:
:+1: